attempt_6_animals/simple-reference-animals.squiggle

```diff
@@ -126,16 +111,260 @@ improvement_as_proportion_of_lifetime = 0.05 to 0.5
 value_fwi_fish = 
   fish_potentially_helped * 
 	improvement_as_proportion_of_lifetime * 
-	(value_happy_salmon_year_in_human_qalys / salmon_estimates.lifetime)
+	(salmon_estimates.value_year_in_human_qalys / salmon_estimates.lifetime)
 
 value_of_shrimp_in_fish = (0.3 to 1)
 // ^ very uncertain, subjective
 
 value_fwi_shrimp = 
   shrimp_potentially_helped * 
 	improvement_as_proportion_of_lifetime * 
-	(value_happy_salmon_year_in_human_qalys / salmon_estimates.lifetime) *
+	(salmon_estimates.value_year_in_human_qalys / salmon_estimates.lifetime) *
   value_of_shrimp_in_fish
 
 value_fwi_so_far = value_fwi_fish + value_fwi_shrimp
-value_fwi_so_far
+proportion_fwi_in_2022 = 1/4 to 1/2
+value_fwi  = value_fwi_so_far * proportion_fwi_in_2022
+fwi_item = {
+  name: "Fish Welfare Inititative (2022)",
+	value: value_fwi -> ss 
+}
+
+/*
+## Open Wing Alliance
+
+Sources: 
+- <https://animalcharityevaluators.org/charity-review/the-humane-league/#comprehensive-review>
+- <https://thehumaneleague.org/article/owa-year-in-review-2022>
+
+What am I estimating: Impact of corporate commitments that OWA made happen in 2022.
+
+Notes on the estimation: The number of corporate campaigns is from OWA's own reporting. The number of chickens affected and the improvement per chicken is particularly speculative here.
+
+Key simplification: I am using the welfare numbers for meat chickens, rather than for egg-laying chicken. This will give inaccurate results, but hopefully not by many orders of magnitude.
+*/
+
+num_corporate_commitments = 150 to 200 
+// ^ "nearly 200 commitments", from https://thehumaneleague.org/article/owa-year-in-review-2022
+average_num_chickens_per_year_per_commitment = 5k to 20k 
```

Comment:
I suggest using the word "mean" instead of "average", to be clearer.

Reply:
Done